src/librustc_codegen_ssa/mir/block.rs

```diff
@@ -147,7 +147,7 @@ impl<'a, 'tcx> TerminatorCodegenHelper<'a, 'tcx> {
                 if bx.cx().tcx().sess.opts.cg.tracer.sir_labels() && bx.cx().has_debug() {
                     let did = fx.instance.def.def_id();
                     let crate_hash = bx.cx().tcx().crate_hash(did.krate).as_u64();
-                    let lbl_name = CString::new(format!("__YK_RET_{}_{}_{}", crate_hash,
+                    let lbl_name = CString::new(format!("__YK_RET_{:x?}_{}_{}", crate_hash,
```

Comment:
Why the `?` in the format string? `u64` should implement `Display`, right?

Reply:
Whoops, that's a typo.